### PR TITLE
Fix Scala Subquery should contain only 1 column

### PR DESCRIPTION
**Pull request by ChobobDev**
resolves : #1062

### CODE REVIEW

**Comment by ChobobDev** [comment]
I think I am ready for the review !

File: core/src/executor/evaluate/error.rs [hunk 1]
```diff
@@ -74,6 +74,9 @@ pub enum EvaluateError {
     #[error("subquery returns more than one row")]
     MoreThanOneRowReturned,
 
+    #[error("subquery recieved more than one column")]
+    MoreThanOneColumnReceived,
```
Comment (devgony) [inline core/src/executor/evaluate/error.rs]: Looks like MoreThanOneColumnReturned sounds more easy to understand.

**Review by devgony — COMMENTED** [review]
Looks like MoreThanOneColumnReturned sounds more easy to understand.

File: core/src/executor/evaluate/error.rs (hunk 1 above)
Comment (ChobobDev) [reply core/src/executor/evaluate/error.rs]: yeah, i totally agree with you,
I have made changes as you guided. Thank you

**Review by ever0de — APPROVED** [review]

**Review by panarch — APPROVED** [review]
Looks nice! Thanks for the contribution 👍

**Pull request merged**